crates/test_utils/src/connector_auth.rs

```diff
@@ -78,7 +79,7 @@ impl ConnectorAuthentication {
     /// Will panic if `CONNECTOR_AUTH_FILE_PATH` env is not set
     #[allow(clippy::expect_used)]
     pub fn new() -> Self {
-        // Do `export CONNECTOR_AUTH_FILE_PATH="/hyperswitch/crates/router/tests/connectors/sample_sample_auth.toml"`
+        // Do `export CONNECTOR_AUTH_FILE_PATH="/hyperswitch/crates/router/tests/connectors/sample_sample_sample_auth.toml"`
```

Comment:
Ummm, why did this happen? There's also another similar change in this file.

Reply:
This is template issue, I am taking this in separate PR